components/ui/button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Button = ({ click, text, size, variant }) => {
```

Comment:
We don't want to pass as pros text and click function, also it's important to create an interface or a type for props to satisfy typescript requirements